lib/core/init.js

```diff
@@ -126,14 +126,16 @@ function allModules(key, checkShouldRun = false) {
 	);
 }
 
-// Adds body classes for enabled options that have `bodyClass: true`
+// Adds body classes for modules or enabled options that have `bodyClass: true`
 // In the form `res-moduleId-optionKey` for boolean options
 // and `res-moduleId-optionKey-optionValue` for enum options
 // spaces in enum option values will be replaced with underscores
-function _addOptionsBodyClasses() {
+function _addModuleBodyClasses() {
 	for (const module of Modules.all()) {
 		if (!Modules.isRunning(module) || _.isEmpty(module.options)) continue;
 
+		if (module.bodyClass) BodyClasses.add(`res-${module.moduleID}`);
```

Comment:
This should happen before the `_.isEmpty(module.options)` check, so it works with modules with no options